lib/radiator_web/views/api/chapters_view.ex

```diff
@@ -0,0 +1,25 @@
+defmodule RadiatorWeb.Api.ChaptersView do
+  use RadiatorWeb, :view
+
+  alias HAL.{Document, Link}
+  alias Radiator.AudioMeta.Chapter
+
+  def render("show.json", assigns) do
+    render(__MODULE__, "chapter.json", assigns)
+  end
+
+  def render("chapter.json", %{conn: conn, chapter: chapter, audio: audio}) do
+    %Document{}
+    |> Document.add_link(%Link{
+      rel: "self",
+      href: Routes.api_audio_chapters_path(conn, :show, audio.id, chapter.id)
+    })
+    |> Document.add_properties(%{
+      id: chapter.id,
```

Comment:
imho this needs at least the `audio_id` as well to be useful for e.g. update, as we need the audio id in the update path too